releasenotes/2.0.3.md

```diff
@@ -0,0 +1,68 @@
+
+Apache jclouds 2.0.3 is the third bugfix release after jclouds 2.0 and comes with several bugfixes and performance improvements. You can [read the official announcement here](https://s.apache.org/jclouds203). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.0.3 new features include:
+
+* Fixed compilation issues in Java 1.8.
+* Completed the provider APIs in the DimensionData provider needed to support the Compute abstraction.
+* Added the Amazon Chine Beijing region.
+* Support `custom-data` and added the `VirtualMachineScaleSet` APIs to Azure ARM.
+* Updated ProfitBricks to the latest API version.
+* Added B2 native prefix and delimiter support.
+
+### Bugs and patches
+
+* [JCLOUDS-831](https://issues.apache.org/jira/browse/JCLOUDS-831) - Use the GroupNamingConvention to generate the names
+* [JCLOUDS-981](https://issues.apache.org/jira/browse/JCLOUDS-981) - Azure: remove mandatory virtual network creation
+* [JCLOUDS-1194](https://issues.apache.org/jira/browse/JCLOUDS-1194) - Java 1.8 compilation issue
+* [JCLOUDS-1255](https://issues.apache.org/jira/browse/JCLOUDS-1255) - Add support for Dimension Data
+* [JCLOUDS-1256](https://issues.apache.org/jira/browse/JCLOUDS-1256) - Inifinite recursion in BoxConfig(File, Image)
+* [JCLOUDS-1258](https://issues.apache.org/jira/browse/JCLOUDS-1258) - Amazon China Beijing Region Support
+* [JCLOUDS-1282](https://issues.apache.org/jira/browse/JCLOUDS-1282) - Support custom-data in Azure ARM template options
+* [JCLOUDS-1314](https://issues.apache.org/jira/browse/JCLOUDS-1314) - B2 native prefix and delimiter support
+* [JCLOUDS-1327](https://issues.apache.org/jira/browse/JCLOUDS-1327) - putBlob with zero length Inputstream is failing on google-cloud-storage
+* [JCLOUDS-1328](https://issues.apache.org/jira/browse/JCLOUDS-1328) - Update ProfitBricks to the latest API version
+* [JCLOUDS-1329](https://issues.apache.org/jira/browse/JCLOUDS-1329) - Azure ARM extraneous resources are not cleaned up on node deletion
+* [JCLOUDS-1332](https://issues.apache.org/jira/browse/JCLOUDS-1332) - BaseComputeService.destroyNodesMatching and BaseComputeService.destroyNode different behavior
+* [JCLOUDS-1342](https://issues.apache.org/jira/browse/JCLOUDS-1342) - Azure-ARM Virtual Machine Scale Set Support
+* [JCLOUDS-1351](https://issues.apache.org/jira/browse/JCLOUDS-1351) - AWS CentOS marketplace AMI misdetecting OS type
+* [JCLOUDS-1353](https://issues.apache.org/jira/browse/JCLOUDS-1353) - AWS EC2 add support for Accelerated computing instance type
+* [JCLOUDS-1358](https://issues.apache.org/jira/browse/JCLOUDS-1358) - putBlob with zero length InputSteam is failing for aws s3
+
+The complete list of fixed issues and improvements can be found [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12341063&styleName=Html&projectId=12314430).
+
+## <a id="knownissues"></a> Known Issues
+
+TODO
```

Comment:
would it be worth to mention Openstack issues here at least? Most of installation are not working now because of keystone v3 missing

Reply:
Sure, it is just the initial version. As I said in the PR description, this is a collaborative work. Please, feel free to directly push the fixes for what you see wrong and add whatever you consider it should be added :)

Reply:
I'd add something like:
```
There are a number of jira issues related to Openstack that are around a better versioning support, namely
JCLOUDS-1197
JCLOUDS-773
JCLOUDS-114

Particularly,  jclouds doesn't support OpenStack Keystone v3 API, currently, but the community is actively working on that -- see https://lists.apache.org/thread.html/0e2df0f9f8168e607d273f3a875576f3c14cf93cf740d8ada1d4e2ef@<dev.jclouds.apache.org>
```
Sounds ok?

Reply:
Lgtm